src/Handshake/ServerNegotiator.php

```diff
@@ -55,20 +55,26 @@ public function handshake(RequestInterface $request) {
             return new Response(400);
         }
 
+        $upgradeSuggestion = [
+            'Connection'             => 'Upgrade',
+            'Upgrade'                => 'websocket',
+            'Sec-WebSocket-Version'  => $this->getVersionNumber(),
+            'Sec-WebSocket-Protocol' => implode(', ', $this->_supportedSubProtocols)
```

Comment:
I think `Sec-WebSocket-Protocol` should only be included if >= 1 supportedSubProtocols

Reply:
New commit coming soon...